frontend/model/database.js

```diff
@@ -16,6 +16,14 @@ const localforage = {
         reject(new Error('Unsupported characters in name: -'))
         return
       }
+      // TODO REMOVEME
+      if (!self.indexedDB) {
+        throw new Error('NO self.indexedDB' + typeof indexedDB)
+      }
```

Comment:
```
Running "exec:flow" (exec) task
Error --------------------------------------------------------------------------------- frontend/model/database.js:21:54

Cannot resolve name `indexedDB`. [cannot-resolve-name]

   21|         throw new Error('NO self.indexedDB' + typeof indexedDB)
                                                            ^^^^^^^^^
```

Reply:
> // TODO REMOVEME

This code isn't meant to be merged (and Flow is wrong, indexedDB is supposed to be defined and typeof is explicitly allowed on undefined names anyhow).

I've added it to debug the iOS indexedDB weirdness.